vhdbuilder/packer/install-dependencies.sh

```diff
@@ -552,6 +552,7 @@ if [ $BCC_EXIT_CODE -eq 0 ]; then
 EOF
 else
   echo "Error: installBcc subshell failed with exit code $BCC_EXIT_CODE" >&2
+  exit $BCC_EXIT_CODE
 fi
 chmod 755 /var/log/bcc_installation.log
```

Comment:
wont we loose the access to this file  ? I though you added that chmod to allow the publishing of the log to ADO

Reply:
good catch! fixed